src/queryweaver/async_client.py

```
@@ -0,0 +1,385 @@
+
+        logging.info("Loading database '%s' from %s", database_name, db_type)
+
+        try:
+            success = await self._load_database_async(database_name, database_url, loader_class)
+            
+            if success:
+                self._loaded_databases.add(database_name)
+                logging.info("Successfully loaded database '%s'", database_name)
+                return True
+            else:
+                raise RuntimeError(f"Failed to load database schema for '{database_name}'")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)